lib/friends/location.rb

```diff
@@ -10,11 +10,13 @@ class Location
     extend Serializable
 
     SERIALIZATION_PREFIX = "- "
+    # NOTE: nickname is the location alias, but `alias` is a keyword
+    NICKNAME_REFIX = "a.k.a. "
```

Comment:
Small typo: `REFIX` -> `PREFIX`

I'd actually prefer to use `alias` everywhere (e.g. `ALIAS_PREFIX`, `alias_str`, `@aliases`, etc.) except where we have to use `nickname` because of the keyword issue. Thoughts?

Reply:
Yes, i find that also better. `nickname` is not used so much so it should make no problem.